src/dispatcher.js

```diff
@@ -99,7 +99,14 @@ class CommandDispatcher {
 	 */
 	async handleMessage(message, oldMessage) {
 		if(!this.shouldHandleMessage(message, oldMessage)) return;
-
+		// Obtain the member if we don't have it
+		if(message.guild && !message.guild.members.has(message.author.id) && !message.webhookID) {
```

Comment:
Why change it from `this.message.channel.type === 'text'`?

Reply:
personal preference I guess. They can be used interchangeably.